### DISCUSSION TITLE
Failed Deploy with 10.1.2 async Span.traceAsyncFn

### DISCUSSION THREAD

**Discussion by mousedown** [vercel/next.js#23582] [Help]
Does anybody know what might be causing vercel to fail deploy using Next.js v 10.1.2? 
I needed to use 10.0.0 for a success. 

`
14:23:52.228 | at /vercel/path0/node_modules/next/dist/export/index.js:31:1106
14:23:52.228 | at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/telemetry/trace/trace.js:5:584)
14:23:52.228 | at async /vercel/path0/node_modules/next/dist/build/index.js:43:49
14:23:52.228 | at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/telemetry/trace/trace.js:5:584)
14:23:52.228 | at async /vercel/path0/node_modules/next/dist/build/index.js:25:1475
14:23:52.228 | at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/telemetry/trace/trace.js:5:584)
14:23:52.270 | error Command failed with exit code 1.
14:23:52.270 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
14:23:52.287 | Error: Command "yarn run build" exited with 1

`

**Comment by sabit990928** [comment]
Have the same issue while using next 10.1.3 version. Occurring on `next build & next export` in my github action
